sql/ddl_internal.sql

```diff
@@ -37,3 +37,33 @@ CREATE OR REPLACE FUNCTION  _timescaledb_internal.chunk_drop_replica(
     node_name               NAME
 ) RETURNS VOID
 AS '@MODULE_PATHNAME@', 'ts_chunk_drop_replica' LANGUAGE C VOLATILE;
+
+CREATE OR REPLACE PROCEDURE _timescaledb_internal.wait_subscription_sync(
+    schema_name    NAME,
+    table_name     NAME,
+    retry_count    INT DEFAULT 18000,
+    retry_delay_ms NUMERIC DEFAULT 0.200
```

Comment:
`pg_sleep` takes seconds as input. So you are basically passing 0.2 seconds as default. Given than retry count is 10000, this will wait for a very long time.

Reply:
This supposed to be an hour in total